Command/PopulateCommand.php

```diff
@@ -90,11 +91,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $index = $input->getOption('index');
         $type = $input->getOption('type');
         $reset = !$input->getOption('no-reset');
+        $delete = !$input->getOption('no-delete');
+
         $options = array(
+            'delete'        => $delete,
```

Comment:
Adding these as `$options` seems to break during the strict validation of options later on https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/Provider/AbstractProvider.php#L65.
`reset` is already there but `delete` is missing.